### ISSUE TITLE
How does it compare to Messenger Desktop?

### ISSUE THREAD

**Issue by ad31mar** [sindresorhus/caprine#1321]
Now that [it's out](https://www.messenger.com/desktop), how does Caprine compare?

**Comment by sindresorhus** [comment]
Caprine is open-source, has privacy features, is more customizable, and also supports Linux.

**Comment by 1nikolas** [comment]
Here is [my opinion](https://github.com/sindresorhus/caprine/issues/1189#issuecomment-607371563) on this. I get the privacy stuff but its running faster and better than Caprine (cause its official)

**Comment by carlinmack** [comment]
I would like someone who is knowledgeable on UWP to chime in on how private the official app must be. Afaik UWP doesn't allow apps to know anything outside of it's container

**Comment by centraleffects** [comment]
> Here is [my opinion](https://github.com/sindresorhus/caprine/issues/1189#issuecomment-607371563) on this. I get the privacy stuff but its running faster and better than Caprine (cause its official)

Does it support Linux? I can't find a way to install it.

**Comment by 1nikolas** [comment]
> > Here is [my opinion](https://github.com/sindresorhus/caprine/issues/1189#issuecomment-607371563) on this. I get the privacy stuff but its running faster and better than Caprine (cause its official)
> 
> Does it support Linux? I can't find a way to install it.

No it doesn't.. For linux Caprine is the only solution

**Comment by johan456789** [comment]
> Here is [my opinion](https://github.com/sindresorhus/caprine/issues/1189#issuecomment-607371563) on this. I get the privacy stuff but its running faster and better than Caprine (cause its official)

I have the opposite experience. The official app is slow on mac. Try cmd + w and then open the app again. It takes 1-3 seconds to do so, while it's instantly with Caprine. The UI is better on Caprine. You can reply to previous messages by clicking a button; you need to right click to access the option in the official app. 

I'll still use Caprine for now.

**Comment by 1nikolas** [comment]
> You can reply to previous messages by clicking a button; you need to right click to access the option in the official app.
You can reply by double clicking outside of the message (on the same row). That's actually kinda smart instead of having lots of buttons. Also you can react by long pressing the message

**Comment by arjitj2** [comment]
I had to switch to the official simply for the lower memory usage. As @1nikolas pointed out it's not Caprine's fault, the official messenger app just gets to use the mobile interface & APIs which are faster.

As I speak right now the official messenger app is using 57 mb RAM, while Caprine is at ~570 mb.

Still love & appreciate Caprine and all the dev work that went into it.

**Comment by botex98** [comment]
Caprine has some features that I like to use, but I switched to the official simply because of two reasons:

1. No JavaScript Exceptions popping up, when using the app normaly. (this happens at least a few times a week)
2. It does not give me an error every 15 minutes that my connection is poor.

But yeah, Caprine was awesome while I use it apart from the two issues.

**Comment by rtpHarry** [comment]
Hmm seems those memory benefits are not translated to the mac version.

The main app is only clocking 68mb but then its gobbling up hundreds more in its support services:

![image](https://user-images.githubusercontent.com/1038062/81168949-e4cda880-8f8f-11ea-9567-73baa78f68c8.png)

**Comment by CvX** [comment]
Messenger is also an Electron app so it's definitely not a silver bullet. And it also has its share of issues. I, personally, can't migrate to it for now.

**Comment by 1nikolas** [comment]
From my experience with the app, I think it uses Electron only for calls. Everything else seems to be native

**Comment by CvX** [comment]
`alt-cmd-i` disagrees 😅 

![image](https://user-images.githubusercontent.com/66961/81172504-a55a8880-8f9e-11ea-859d-b8b43a8ae001.png)

**Comment by 1nikolas** [comment]
Hmm they could have made it native. Anyway point is that it doesn't use the website directly so it doesn't lag that much like Caprine. But still it has some bugs

**Comment by rtpHarry** [comment]
Left it running for half a day and its now sitting on 1gb of ram:

![image](https://user-images.githubusercontent.com/1038062/81237562-e337cb00-8ff7-11ea-8a7a-90d7e22afb9d.png)

**Comment by 1nikolas** [comment]
@rtpHarry Caprine has the same problem. Caprine and Messenger are essentially an html page which never refreshes, so everything you do (send/recieve messages, load images, or maybe going a few messages back) is loaded on the ram. Restarting the app is a quick and easy fix

**Comment by rtpHarry** [comment]
fair enough... I guess I'm just a bit rooting for it to suck as caprine has had so much development and i don't want it to be bested :P 

in just over one day its gobbled up 3gb of ram. I have noticed Caprine chugging ram but no more than anything chrome related. This seems like a bug to me, but I'm not an expert:

![image](https://user-images.githubusercontent.com/1038062/81339420-133da780-90a6-11ea-8550-bf5313637fb4.png)

**Comment by 1nikolas** [comment]
> fair enough... I guess I'm just a bit rooting for it to suck as caprine has had so much development and i don't want it to be bested :P 
> 
> in just over one day its gobbled up 3gb of ram. I have noticed Caprine chugging ram but no more than anything chrome related. This seems like a bug to me, but I'm not an expert:
> 
> ![image](https://user-images.githubusercontent.com/1038062/81339420-133da780-90a6-11ea-8550-bf5313637fb4.png)
> 

Hmm you might be right

**Comment by 1nikolas** [comment]
Okay its been a few months using the official app and I have to say that it has become worse. It lags a lot and it freezes most of the time for like a second periodically (despite me having a better PC than I used to have). I don't know how they managed to make it worse, but they did. So I'm switching back to Caprine

**Comment by matthewstrasiotto** [comment]
> Here is [my opinion](https://github.com/sindresorhus/caprine/issues/1189#issuecomment-607371563) on this. I get the privacy stuff but its running faster and better than Caprine (cause its official)

I disagree on the point of "better" - Caprine has more features & a more responsive layout.

It's now been a while since I used native, but Caprine being electron powered definitely does take time to start up, & gobbles a bit of memory- I have 16 to 64gbs on my workstations, and if you have 16GB, you're probably going to be fine (It might be an issue on my old surface pro, I guess).

That said, I remember there being problems with the native app (I just don't remember what they were)

**Comment by 1nikolas** [comment]
> > Here is [my opinion](https://github.com/sindresorhus/caprine/issues/1189#issuecomment-607371563) on this. I get the privacy stuff but its running faster and better than Caprine (cause its official)
> 
> I disagree on the point of "better" - Caprine has more features & a more responsive layout.
> 
> It's now been a while since I used native, but Caprine being electron powered definitely does take time to start up, & gobbles a bit of memory- I have 16 to 64gbs on my workstations, and if you have 16GB, you're probably going to be fine (It might be an issue on my old surface pro, I guess).
> 
> That said, I remember there being problems with the native app (I just don't remember what they were)

Look on the comment above yours

**Comment by CaptainYesz** [comment]
I found Caprine chugging 6gb of memory and heavy disk usage so back to official for me

I tend to copy paste images into my chats... combined with me keeping it open 24/7 on my desktop

**Comment by pktiuk** [comment]
This RAM issue was already mentioned here: https://github.com/sindresorhus/caprine/issues/1244  
It was the main reason I stopped using Caprine.  
I think it should be a top priority issue for now.

**Comment by mzso** [comment]
I just got offered the desktop messenger. Was it not available for Windows before?

Is it any good? Caprine right now has issues due to the re-design of messenger.com. It tends to hang and not update anymore. And there are no alerts.

**Comment by chris-larsen** [comment]
I noticed the native messenger app for Mac has completely broken search. It will hang and freeze. If I try to scroll up enough in a chat, it'll eventually just leave me with a spinning wheel. This doesn't happen when using the web messenger, and ultimately I'm using Caprine because the search/indexing is just borked on facebooks native mac app. Also Facebook still hasn't released an update that follows the UI guidelines of Big Sur, and the corners of the window aren't rounded lol. Even though Caprine is a little slower then the FB native Mac app, at least Caprine is getting updates. It doesn't feel like Facebooks put an update into their Mac app forever.

**Comment by binarykitchen** [comment]
But then why am I still seeing suggestions and ads on Caprine to download Messenger Desktop? Don't want to see these.

**Comment by mquevill** [comment]
> But then why am I still seeing suggestions and ads on Caprine to download Messenger Desktop? Don't want to see these.

Where are you seeing these suggestions/ads? Version 2.53.0 removed the banner at the bottom of the conversation list and in the preferences modal window (#1621). If you could include screenshots and system info, that would help us track down these objects (and remove them).

**Comment by sheldonhull** [comment]
I had to flip back to Caprine as the official app doesn't seem to support logging in via Messenger only account without facebook. The desktop app reactivated my facebook account twice and deactivating seemed to log out the app. 

Caprine supports a deactivated facebook account with 2FA without requiring my facebook account to be active.

**Comment by EldrinSMP** [comment]
Sorry to necro this thread, but it seemed like the best place to ask my question. Does Caprine support E2E Encryption like the desktop app?

**Comment by grzegorj8** [comment]
Messenger Desktop (on PC) refuses to send files or even images, while Caprine has no problems with them. On the other hand, Caprine cannot write anything, the only possibility is to use a browser (which is very uncomfortable). As long as the saving function is not added to Caprine, I will have to use both programs at the same time. It is frustrating...
